docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 services:
   postgres:
     image: postgres:15.6
-    command: postgres -c 'max_connections=200'
+    command: postgres -c 'max_connections=500'
```

Comment:
this may be a symptom of a different problem, but i need to keep bumping this value